src/main/kotlin/org/rust/ide/formatter/impl/utils.kt

```diff
@@ -48,24 +47,22 @@ val MACRO_ARGS = ts(MACRO_ARG, FORMAT_MACRO_ARGS, TRY_MACRO_ARGS)
 
 val FN_DECLS = ts(FN_ITEM, FOREIGN_FN_DECL, TRAIT_METHOD_MEMBER, IMPL_METHOD_MEMBER, BARE_FN_TYPE, LAMBDA_EXPR)
 
-val FN_SHARED_ALIGN_OWNERS = orSet(PARAMS_LIKE, ts(RET_TYPE, WHERE_CLAUSE))
-
 
 val PsiElement.isTopLevelItem: Boolean
-    get() = (this is RustItemElement || this is RustAttrElement) && this.parent is RustFile
+    get() = (this is RustItemElement || this is RustAttrElement) && parent is RustMod
```

Comment:
Yeah, I haven't noticed this. It was correct when parent could be only a file. Fix it here?